docs/docs/advanced/real-time-games.md

```diff
@@ -52,6 +52,67 @@ Rune.initLogic({
 
 ```
 
+## World time {#world-time}
+
+To track time outside the game, Rune exposes `Rune.worldTime()`. This function returns a timestamp in milliseconds since the epoch. Using this value allows to build daily challenges, time-based events. `Rune.worldTime()` has 1 second precision.
```

Comment:
```suggestion
To track time outside the game, Rune exposes `Rune.worldTime()`. This function returns a timestamp in milliseconds since the epoch. Using this value allows building daily challenges and time-based events. `Rune.worldTime()` has 1 second precision.
```